test/powershell/engine/ETS/Adapter.Tests.ps1

```diff
@@ -112,6 +112,27 @@ Describe "Adapter Tests" -tags "CI" {
             Remove-TypeData TestCodeMethodInvokationWithVoidReturn
         }
     }
+
+    It "Count and length property works for singletons" {
+        $x = 5
+        $x.Count | Should Be 1
+        $x.Length | Should Be 1
+
+        $null.Count | Should Be 0
+        $null.Length | Should Be 0
+
+        (10).Count | Should Be 1
+        (10).Length | Should Be 1
+
+        ("a").Count | Should Be 1
+        ("a").Length | Should Be 1
```

Comment:
The `Length` property exists for strings, so this isn't (shouldn't be) testing singletons.

Reply:
Fixed.